R/drive_publish.R

```diff
@@ -26,150 +32,119 @@ drive_publish <- function(file, ..., verbose = TRUE) {
 
 #' Unpublish Google Drive file
 #'
-#' @template file
-#' @param ... Name-value pairs to add to the API request body.
-#' @template verbose
+#' Unpublish a Google Drive file to the web. You can only publish "Google" files, such
+#' as Google Documents, Google Spreadsheets, or Google Slides, therefore you cannot
+#' (and there would be no need to) unpublish non-"Google" type files.
```

Comment:
`drive_publish()` and `drive_unpublish()` seem like good candidates to document together. Then you don't have to repeat this blurb, the examples are easier to write, etc.